src/Pages/Signup/SignupSelect.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+
+class SignupSelect extends Component {
+  render() {
+    const { name, options, handleChange } = this.props;
+    return (
+      <select
+        name={name}
+        onChange={e => handleChange(e)}
+        className="signupSelect"
+      >
+        {options.map((el, idx) => (
+          <>
+            <option value={el.id} key={idx}>
+              {el.option}
```

Comment:
```suggestion
        {options.map((el, idx) => (
          <Fragment key={idx}>
            <option value={el.id}>
              {el.option}
```
공식문서에서 React.Fragment 찾아보고 공부해두세요!